public/app/plugins/datasource/prometheus/components/monaco-query-field/monaco-completion-provider/completions.ts

```diff
@@ -23,6 +23,9 @@ type Metric = {
 export type DataProvider = {
   getHistory: () => Promise<string[]>;
   getAllMetricNames: () => Promise<Metric[]>;
+  // these mirror the prometheus endpoints
```

Comment:
I think don't understand this comment. All `getAllLabelNames`, `getLabelValues` and `getSeries` mirror prometheus endpoints? `getAllMetricNames` doesn't? And what does it mean?

Reply:
sorry, i wanted to explain that those 3 are basically connecting to the similarly-named prometheus endpoints.

getAllMetricNames is different, because that one needs both metric-names and meta-info about those metric-names and merge them...

but probably not that important to have the comment there, so i'll remove it for now.